Packs/PAN-OS/Integrations/Panorama/README.md

```diff
@@ -4687,3 +4687,558 @@ Installs specific content update file.
 >|JobID|Status|
 >|---|---|
 >| 30 | Pending |
+
+
+
+### pan-os-platform-get-arp-tables
+***
+Gets all arp tables from all firewalls in the topology.
+
+
+#### Base Command
+
+`pan-os-platform-get-arp-tables`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| device_filter_string | String to filter to only show specific hostnames or serial numbers. | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| PANOS.ShowArp.Summary.hostid | string | ID of PAN-OS host | 
+| PANOS.ShowArp.Summary.max | string | Maximum supported ARP Entries | 
+| PANOS.ShowArp.Summary.total | string | Total current arp entries | 
+| PANOS.ShowArp.Summary.timeout | string | ARP entry timeout | 
+| PANOS.ShowArp.Summary.dp | string | Firewall dataplane associated with Entry | 
+| PANOS.ShowArp.Result.hostid | string | ID of PAN-OS host | 
+| PANOS.ShowArp.Result.interface | string | Network interface learnt ARP entry | 
+| PANOS.ShowArp.Result.ip | string | layer 3 address | 
+| PANOS.ShowArp.Result.mac | string | Layer 2 address | 
+| PANOS.ShowArp.Result.port | string | Network interface matching entry | 
+| PANOS.ShowArp.Result.status | string | ARP Entry status | 
+| PANOS.ShowArp.Result.ttl | string | Time to Live | 
+
+#### Command example
+```!pan-os-platform-get-arp-tables```
+#### Context Example
+```json
+{
+    "PANOS": {
+        "ShowArp": {
+            "Result": [
+                {
+                    "hostid": "0111112222333444",
+                    "interface": "ethernet1/1",
+                    "ip": "192.168.1.1",
```

Comment:
make sure there isn't any sensitive data that will be part of the README such as IPs/fqdn and so on.

Reply:
I slightly munged some of these IPs - they are all internal to my lab so no risk of secrets in there, but its a bit easier to add them to .secrets-ignore this way.